src/Common/tests/System/Net/Http/LoopbackServer.cs

```diff
@@ -153,6 +153,7 @@ private static string ComputeWebSocketHandshakeSecurityAcceptValue(string secWeb
         {
             options = options ?? new Options();
             Socket s = await server.AcceptAsync().ConfigureAwait(false);
+            s.NoDelay = true;
```

Comment:
Did this affect something tangibly?

Reply:
No, but it seems pretty broken to not set NoDelay.

Reply:
I'm fine with seeing it, but why is that "broken"? Just curious.

Reply:
I should clarify: Without NoDelay, the *dribble tests* are pretty broken.  The kernel will buffer sends so we won't actually send the bytes one at a time, which is the point of the dribble test stuff.

The loopback server is not broken in general, though it does seem slightly better to use NoDelay, just because kernel buffering may produce different behavior than we're expecting.

Reply:
Ok